docs/samples/encrypted-notes.md

```diff
@@ -86,38 +84,22 @@ Once authenticated with II:
 Follow the steps below to deploy this sample project.
 
 ## Prerequisites
-- [x] Install the [IC SDK](../developer-docs/setup/install/index.mdx).
+- [x] Install the [IC SDK](../developer-docs/setup/install).
 - [x] Download and install [Docker](https://docs.docker.com/get-docker/) if using the Docker option. 
-- [x] Download the GitHub repo containing this project's files: https://github.com/dfinity/examples/tree/master/motoko/encrypted-notes-dapp. (If using Rust, use the /master/rust/encrypted-notes-dapp folder.)
+- [x] Download the GitHub repo containing this project's files: https://github.com/dfinity/examples/tree/master/motoko/encrypted-notes-dapp.
 
 ### Step 1. Navigate inside of the project's folder:
 
 ```
 cd examples/motoko/encrypted-notes-dapp
 ```
 
-or
-
-```
-cd examples/rust/encrypted-notes-dapp
-```
 
 ### Step 2: Set an environmental variable reflecting which backend canister you'll be using:
-For Motoko deployment run:
 
 ```
 export BUILD_ENV=motoko
 ```
-
-For Rust deployment run:
-
-```
-export BUILD_ENV=rust
-```
-
-:::info
-Building the Rust canister requires either the Rust toolchain installed on your system or Docker-backed deployment (see below).
-:::
```

Comment:
why do you suggest to delete the Rust deployment part? The code is in the samples repo and works fine, afaics...